.github/script/get-prev-version

```diff
@@ -1,11 +1,26 @@
-#!/usr/bin/env ruby
+#!/usr/bin/env node
 
-versions = `npm view amazon-chime-sdk-js versions --json`.strip.split("\n")
-# The output would be something like
-#[
-#  "1.0.0",
-#  ...
-#  "2.8.0", <- previous version
-#  "2.9.0"  <- latest version
-#]
-puts versions[versions.size-3][3..-3]
\ No newline at end of file
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const exec = require('child_process').execSync;
+
+const versions = exec('npm view amazon-chime-sdk-js versions --json').toString().trim().split("\n");
+// The output would be something like
+// [
+//   "1.0.0",
+//       ...
+//   "2.8.0", <- previous version
+//   "2.9.0"  <- latest version
+// ]
+// exec('npm view amazon-chime-sdk-js versions --json', (err, stdout, _) => {
```

Comment:
Want to remove these comments :) and the ruby one below?

Reply:
Oops :D